README.md

```diff
@@ -1,2 +1,15 @@
 # Smashing-Magazine
-PROJECT: DESIGN TEARDOWN HTML and CSS
+<h1>PROJECT: DESIGN TEARDOWN HTML and CSS</h1>
+<ul>
+<li>
```

Comment:
While you definitely can write `html` inside markdown files, it's always better to use the [markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). Markdown syntax is really easy to learn.